### PR TITLE
Load forecasting front end

### PR DESCRIPTION
**Pull request by derins**
### Why:
* New code received for loadforecasting algs + required frontend changes.

### What:
* Updated the backend code for EVLoadModel.py and configurations.py
* frontend updated

Screen Shot 2020-10-26 at 12 19 18 PM

### CODE REVIEW

File: frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js [hunk 1]
```diff
@@ -143,7 +146,9 @@ class AlgInputsLoadForecast extends Component {
                 res_l2_smooth: this.state.res_l2_smooth,
                 week_day: this.state.week_day,
                 publicl2_daily_use: parseFloat(this.state.publicl2_daily_use),
-                mixed_batteries: this.state.mixed_batteries,
+                small_batt: parseFloat(this.state.small_batt),
+                big_batt: parseFloat(this.state.big_batt),
+                all_batt: parseFloat(this.state.all_batt),
```
Comment (jongoncalves) [inline frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js]: You should really camelCase for your naming convention in JS. Snake casing is for python....

**Review by jongoncalves — APPROVED** [review]

File: frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js (hunk 1 above)
Comment (derins) [reply frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js]: You're right - fixed

File: frontend/src/Components/Api/algorithmDefaultParams.js [hunk 2]
```diff
@@ -28,7 +28,9 @@ export const loadForecastDefaultParams = {
     res_l2_smooth: true,
     week_day: true,
     publicl2_daily_use: 0.33,
-    mixed_batteries: "None",
+    small_batt: 0,
```
Comment (avpeery) [inline frontend/src/Components/Api/algorithmDefaultParams.js]: heads up on changing these variable names if the snake case was replaced with camel case, as per Jonathan's comment

Comment (derins) [reply frontend/src/Components/Api/algorithmDefaultParams.js]: Most of these were already snake case when we got the app. I'll go ahead and update these also.

Comment (derins) [reply frontend/src/Components/Api/algorithmDefaultParams.js]: fixed

**Review by avpeery — APPROVED** [review]

**Pull request merged**